cpp/src/aztec/honk/composer/composer_helper/standard_honk_composer_helper.hpp

```diff
@@ -26,34 +28,31 @@ template <typename CircuitConstructor> class ComposerHelper {
     // TODO(kesha): we need to put this into the commitment key, so that the composer doesn't have to handle srs at all
     std::shared_ptr<bonk::ReferenceStringFactory> crs_factory_;
     bool computed_witness = false;
-    ComposerHelper()
-        : ComposerHelper(
+    StandardHonkComposerHelper()
+        : StandardHonkComposerHelper(
               std::shared_ptr<bonk::ReferenceStringFactory>(new bonk::FileReferenceStringFactory("../srs_db/ignition")))
     {}
-    ComposerHelper(std::shared_ptr<bonk::ReferenceStringFactory> crs_factory)
+    StandardHonkComposerHelper(std::shared_ptr<bonk::ReferenceStringFactory> crs_factory)
         : crs_factory_(std::move(crs_factory))
     {}
 
-    ComposerHelper(std::unique_ptr<bonk::ReferenceStringFactory>&& crs_factory)
+    StandardHonkComposerHelper(std::unique_ptr<bonk::ReferenceStringFactory>&& crs_factory)
         : crs_factory_(std::move(crs_factory))
     {}
-    ComposerHelper(std::shared_ptr<bonk::proving_key> p_key, std::shared_ptr<bonk::verification_key> v_key)
+    StandardHonkComposerHelper(std::shared_ptr<bonk::proving_key> p_key, std::shared_ptr<bonk::verification_key> v_key)
         : circuit_proving_key(std::move(p_key))
         , circuit_verification_key(std::move(v_key))
     {}
-    ComposerHelper(ComposerHelper&& other) noexcept = default;
-    ComposerHelper(const ComposerHelper& other) = delete;
-    ComposerHelper& operator=(ComposerHelper&& other) noexcept = default;
-    ComposerHelper& operator=(const ComposerHelper& other) = delete;
-    ~ComposerHelper() = default;
+    StandardHonkComposerHelper(StandardHonkComposerHelper&& other) noexcept = default;
+    StandardHonkComposerHelper(const StandardHonkComposerHelper& other) = delete;
+    StandardHonkComposerHelper& operator=(StandardHonkComposerHelper&& other) noexcept = default;
+    StandardHonkComposerHelper& operator=(const StandardHonkComposerHelper& other) = delete;
+    ~StandardHonkComposerHelper() = default;
 
     std::shared_ptr<bonk::proving_key> compute_proving_key(const CircuitConstructor& circuit_constructor);
     std::shared_ptr<bonk::verification_key> compute_verification_key(const CircuitConstructor& circuit_constructor);
 
-    void compute_witness(const CircuitConstructor& circuit_constructor)
-    {
-        compute_witness_base<program_width>(circuit_constructor);
-    }
+    void compute_witness(const CircuitConstructor& circuit_constructor) { compute_witness_base(circuit_constructor); }
```

Comment:
Is it possible to simplify the chain of `compute_witness` functions here and for the corresponding standard plonk composer helper? It seems like we could eliminate the existing `compute_witness` then rename `compute_witness_base` to `compute_witness` since `standard_honk_composer_helper` defines both of them and I wouldn't think anything will ever inherit from `standard_honk_composer_helper`. Then in addition we could rename `compute_witness_base_common` to just `compute_witness_base`?

Reply:
Sounds reasonable

Reply:
Updated